docs/querying.md

```diff
@@ -16,6 +16,15 @@ Post.where(:created_at, :gt, Time.local - 7.days)
 
 Supported operators are :eq, :gteq, :lteq, :neq, :gt, :lt, :nlt, :ngt, :ltgt, :in, :nin, :like, :nlike
 
+Alternatively, where accepts a raw SQL clause, with "?" as a placeholder to avoid SQL Injection.
```

Comment:
```suggestion
Alternatively, `#where` and `#or` accept a raw SQL clause, with "?" as a placeholder to avoid SQL Injection.
```

Since `?` is not a valid placeholder in all databases, I would suggest either rewording the sentence to specify that the example is for MySQL/Sqlite, and ofc wouldn't work in Postgres.

Reply:
The implementation actually swaps out any `?` placeholders using the `add_parameter` method, so in the case of MySQL and SQLite it'll leave in `?`, but when using the Postgres adapter it'll replace it properly with `$n`. Not sure how to link it here, but you can see the difference if you compare the spec I added for MySQL/SQLite with the Postgres one.

Maybe I should make this clearer though, in the documentation?

Reply:
Just updating the example to have placeholders specific to PG would be good enough I think.  Could possibly reword the sentence to not use a specific symbol, but just say something like `using placeholders related to the DB you're using`.

Reply:
Hmm, but the "correct" placeholder to use is indeed `?` for all databases, because (at least for Postgres) the builder needs to assign the placeholders sequentially. This also keeps that aspect abstracted from the user. 

To be honest, I pretty much only now realized Postgres doesn't seem to use `?`, but I'm not sure what the alternative would be - maybe using just `$`, and having it replace that instead for just Postgres?